src/rpdk/core/project.py

```diff
@@ -275,7 +275,7 @@ def validate_and_load_resource_settings(self, raw_settings):
         self.entrypoint = raw_settings["entrypoint"]
         self.test_entrypoint = raw_settings["testEntrypoint"]
         self.executable_entrypoint = raw_settings.get("executableEntrypoint")
-        self._plugin = load_plugin(raw_settings["language"])
+        # self._plugin = load_plugin(raw_settings["language"])
```

Comment:
?  Whats the impact to this or why do we need to do this?

Reply:
my bad, this is just for my local setup. I included it by mistake